syft/frameworks/torch/hook/hook.py

```diff
@@ -213,11 +213,6 @@ def __init__(
         # Hook torch.optim (containing optim.SGD, Adam, etc)
         self._hook_optim()
 
-        # Hook the Crypten module
-        if dependency_check.crypten_available:
```

Comment:
Here we should have those lines, only that we should call the ```hook_crypten_module``` that is defined in the other file.

Also, in this file, we should remove the ```_hook_crypten``` and ```_hook_crypten_module``` methods